core/serialization/download_xlsx_serializer.py

```diff
@@ -38,8 +38,8 @@ def serialize_xlsx_data(programmes, programme_outcomes, projects, project_outcom
     programme_outcomes = [
         OutcomeDataSerializer(
             outcome_data,
-            programme_name_mapping[outcome_data.programme.programme_id],
-            organisation_mapping[outcome_data.programme.programme_id],
+            programme_name_mapping.get(outcome_data.programme.programme_id, None),
```

Comment:
Do we need the `None` second argument? Is that not the default behaviour of `.get`?

Reply:
No we don't, I forgot to remove after checking